app/views/discovered_hosts/_discovered_host.html.erb

```diff
@@ -7,6 +7,8 @@
 <td class="hidden-tablet hidden-xs"><%= discovery_attribute(host, :disk_count) %></td>
 <td class="hidden-tablet hidden-xs"><%= number_to_human_size(discovery_attribute(host, :disks_size, 0) * 1024 * 1024) %></td>
 <% if Setting['discovery_fact_column'].present? %>
-  <td class="hidden-tablet hidden-xs"><%= host.facts_hash[Setting['discovery_fact_column']] || 'N/A' %></td>
+<% Setting['discovery_fact_column'].to_s.split(",").each do |fact_column| %>
```

Comment:
> I don't think it is more vulnerable than it previously was. At worse, it will just display an empty column(s) since the fact(s) cannot be found.

Ohad is right, ideally there is a method with rescue catching all
exceptions returning empty array.

-- 
Later,
 Lukas #lzap Zapletal